service/worker/batcher/batcher.go

```diff
@@ -76,6 +80,7 @@ func (s *Batcher) Start() error {
 	ctx := headers.SetCallerInfo(context.Background(), headers.SystemBackgroundCallerInfo)
 	workerOpts := worker.Options{
 		BackgroundActivityContext: ctx,
+		Identity:                  "temporal-system@" + s.hostInfo.Identity(),
```

Comment:
Trying to remember why we're putting the host identity here. Is it just for our own debugging purposes? This will be visible to Temporal Cloud customers and may not be desirable since it's exposing implementation details (given that this isn't exactly sensitive information).

Reply:
Yes, this is more context for debugging. I understand the footprinting concern. We discussed this briefly and the consensus was it was of marginal concern.